src/neuroconv/datainterfaces/ecephys/openephys/openephyslegacydatainterface.py

```diff
@@ -0,0 +1,73 @@
+"""Authors: Szonja Weigl, Cody Baker."""
+from typing import Optional, List
+
+from ..baserecordingextractorinterface import BaseRecordingExtractorInterface
+from ....utils import FolderPathType, get_schema_from_method_signature
+
+
+class OpenEphysLegacyRecordingInterface(BaseRecordingExtractorInterface):
+    """Primary data interface for converting legacy Open Ephys data (.continuous files).
+    Uses :py:class:`~spikeinterface.extractors.OpenEphysLegacyRecordingExtractor`."""
+
+    @classmethod
+    def get_source_schema(cls):
+        """Compile input schema for the RecordingExtractor."""
+        source_schema = super().get_source_schema()
+        source_schema["properties"]["folder_path"][
+            "description"
+        ] = "Path to directory containing OpenEphys legacy files."
+
+        return source_schema
+
+    def __init__(
+        self,
+        folder_path: FolderPathType,
+        stream_name: Optional[str] = None,
+        verbose: bool = True,
+    ):
+        """
+        Initialize reading of OpenEphys legacy recording (.continuous files).
+        See :py:class:`~spikeinterface.extractors.OpenEphysLegacyRecordingExtractor` for options.
+
+        Parameters
+        ----------
+        folder_path : FolderPathType
+            Path to OpenEphys directory.
+        stream_name : str, optional
+            The name of the recording stream.
+            When the recording stream is not specified the channel stream is chosen if available.
+            When channel stream is not available the name of the stream must be specified.
+        verbose : bool, default: True
+        """
+
+        available_streams = self.get_stream_names(folder_path=folder_path)
+        stream_name = stream_name or "Signals CH"
```

Comment:
```suggestion
        stream_name: str = "Signals CH",
        verbose: bool = True,
    ):
        """
        Initialize reading of OpenEphys legacy recording (.continuous files).
        See :py:class:`~spikeinterface.extractors.OpenEphysLegacyRecordingExtractor` for options.

        Parameters
        ----------
        folder_path : FolderPathType
            Path to OpenEphys directory.
        stream_name : str, defulat: "Signals CH"
            The name of the recording stream.
        verbose : bool, default: True
        """

        available_streams = self.get_stream_names(folder_path=folder_path)
```

Is there something wrong with this solution?